index-api/src/Ergvein/Index/API/V1.hs

```diff
@@ -2,10 +2,35 @@ module Ergvein.Index.API.V1 where
 
 import Servant.API
 import Servant.API.Generic
-import Ergvein.Index.API.Models
+import Ergvein.Types.Currency
+import Ergvein.Index.API.Types
 
-type IndexGetBalance = QueryParam "lockingScriptHash" LockingScriptHash :> Get '[JSON] BalanceResponce
+type Body a = ReqBody '[JSON] a
+type GetResp a = Get '[JSON] a
+type PostResp a = Post '[JSON] a
 
+
+type IndexGetBalance = QueryParam "pubKeyScriptHash" PubKeyScriptHash :> GetResp BalanceResponse
```

Comment:
We need a url postfix for each method or server isn't able to distinguish incoming requests. Like 
``` haskell
type IndexGetBalance = "balance" :> QueryParam "pubKeyScriptHash" PubKeyScriptHash :> GetResp BalanceResponse
```